### PR TITLE
New Topic: Cybersecurity

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Created a new topic Cybersecurity with a placeholder entry for the various new cybersecurity career entries that will be forthcoming, as well as a concept for Nmap, which will house new term entries for Nmap.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cybersecurity/concepts/nmap/terms/nmap-scan/nmap-scan.md
Comment (Dusch4593) [inline content/cybersecurity/concepts/nmap/terms/nmap-scan/nmap-scan.md:16]:
```suggestion
The most basic type of **scan** is done with the cli command `nmap` with just a target specified. The target is the IP address or domain name being scanned. It should be a domain the user owns or one the user has written permission to scan.
```

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM! 👍🏻

Just had one tiny suggestion.

**Comment by SSwiniarski** [comment]
@Dusch4593, I implemented your suggestion.

File: content/cybersecurity/concepts/nmap/terms/nmap-scan/nmap-scan.md
Comment (KTom101) [inline content/cybersecurity/concepts/nmap/terms/nmap-scan/nmap-scan.md:26]:
```suggestion
Nmap divides ports into one of six states:
```

File: content/cybersecurity/concepts/nmap/terms/nmap-scan/nmap-scan.md
Comment (KTom101) [inline content/cybersecurity/concepts/nmap/terms/nmap-scan/nmap-scan.md:27]: Since these are states, I feel they should each be back-ticked.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski I've added some changes to this.

**Comment by SSwiniarski** [comment]
@KTom101, I implemented your suggestions.

File: content/cybersecurity/concepts/hacking/hacking.md
Comment (HishamT) [inline content/cybersecurity/concepts/hacking/hacking.md:57]: No Codecademy Doc for ransomware attack?

**Review by HishamT — APPROVED** [review]
Left some minor comments. Otherwise, looks good!

File: content/cybersecurity/concepts/hacking/hacking.md
Comment (KTom101) [reply content/cybersecurity/concepts/hacking/hacking.md:57]: @HishamT Perhaps we should add it to the issues tab? What do you think @SSwiniarski?

**Review by KTom101 — APPROVED** [review]
@SSwiniarski LGTM 👍

File: content/cybersecurity/concepts/hacking/hacking.md
Comment (SSwiniarski) [reply content/cybersecurity/concepts/hacking/hacking.md:57]: @KTom101, @HishamT  It's a good topic for Cybersecurity once we merge this.

**Comment by SSwiniarski** [comment]
@HishamT, I implemented your change.

**Comment by SSwiniarski** [comment]
@HishamT, @Dusch4593, @KTom101, any reason we can't merge this now?

**Review by Dusch4593 — APPROVED** [review]

**Comment by KTom101** [comment]
@SSwiniarski I merged it. I made a tiny edit where I noticed "Nmap" wasn't capitalized ([3404227](https://github.com/Codecademy/docs/pull/1189/commits/3404227b77fb11c2882c86cd0fda15006c7e4f74)). Otherwise, it looked good to go 🔥

Live Links

HubPage: https://www.codecademy.com/resources/docs/cybersecurity
Cyber Attack: https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack
Hacking: https://www.codecademy.com/resources/docs/cybersecurity/hacking
Multi-Factor Authentication:
Nmap: https://www.codecademy.com/resources/docs/cybersecurity/nmap


Thanks, @Dusch4593 @HishamT for reviewing!

**Pull request merged**
